presto-product-tests/bin/run_on_docker.sh

```diff
@@ -142,12 +142,16 @@ fi
 # catch terminate signals
 trap terminate INT TERM EXIT
 
-if [[ "$ENVIRONMENT" == "singlenode" || "$ENVIRONMENT" == "multinode" ]]; then
-  EXTERNAL_SERVICES="hadoop-master mysql postgres cassandra"
-elif [[ "$ENVIRONMENT" == "singlenode-sqlserver" ]]; then
+if [[ "$ENVIRONMENT" == "singlenode-sqlserver" ]]; then
   EXTERNAL_SERVICES="hadoop-master sqlserver"
 elif [[ "$ENVIRONMENT" == "singlenode-ldap" ]]; then
   EXTERNAL_SERVICES="hadoop-master ldapserver"
+elif [[ "$ENVIRONMENT" == "singlenode-mysql" ]]; then
+  EXTERNAL_SERVICES="hadoop-master mysql"
```

Comment:
do we need hadoop when testing mysql?

Reply:
We don't. But due to the limitation in Tempto related to convention based tests, no matter what test suited you select it will alwasy try to provision TPCH dataset in Hive. We should fix that as part of the followup.

Reply:
Can you file a ticket for tempto for that? I will try to fix that.

Reply:
https://github.com/prestodb/tempto/issues/263